src/main/java/org/folio/processor/translations/CustomDateParseException.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.processor.translations;
+
+public class CustomDateParseException extends RuntimeException {
+
+  public CustomDateParseException(String message) {
```

Comment:
I would suggest using an Error Code class , to do it right the first time

Reply:
done